base_tier_validation/models/tier_validation.py

```diff
@@ -34,7 +34,6 @@ class TierValidation(models.AbstractModel):
         inverse_name="res_id",
         string="Validations",
         domain=lambda self: [("model", "=", self._name)],
-        auto_join=True,
```

Comment:
Why removing this?

Reply:
This is done for performance reason and it's not related with the change of the field type. With this sentence, you allow the ORM to join in one SQL fetchings of both tables without caring about the specific security rules of the children table, which is correct in this case AFAIK.